ui/assets/icons/icons.mjs

```diff
@@ -0,0 +1,56 @@
+export default {
```

Comment:
no way to auto-generate this?

Reply:
Maybe in the future, but I think this is not too important right now.